plugins/PreviewStage/PreviewMain.qml

```diff
@@ -12,14 +12,26 @@ import Cura 1.0 as Cura
 Item
 {
 
+    // An Item whose bounds are guaranteed to be safe for overlays to be placed.
+    // Defaults to parent, ie. the entire available area
+    property var safeArea: parent
+
     // Subtract the actionPanel from the safe area. This way the view won't draw interface elements under/over it
-    Item {
-        id: safeArea
-        visible: false
-        anchors.left: parent.left
-        anchors.right: actionPanelWidget.left
-        anchors.top: parent.top
-        anchors.bottom: actionPanelWidget.top
+    Rectangle
+    {
+        id: childSafeArea
+        x: safeArea.x - parent.x
+        y: safeArea.y - parent.y
+        width: actionPanelWidget.x - x
+        height: actionPanelWidget.y - y
+        visible: false // true for debug only
+        color:"#800000FF"
+
+        Component.onCompleted: {
+            print("parent", parent.x, parent.y)
+            print("parent safe", safeArea.x, safeArea.y)
+            print("previewmain safe", childSafeArea.x, childSafeArea.y, childSafeArea.width, childSafeArea.height)
+        }
```

Comment:
This Rectangle needs to be removed.

Reply:
It is used in line 47. If you object to a Rect it can be an Item, but I'd say that an invisible Rectangle is not more expensive than an invisible Item in production code

Reply:
I should remove the print statements tho

Reply:
An invisible rectangle is slightly more expensive than an Item (Although it's not a whole lot)

Reply:
Allright, I'll replace them